venue.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>WiDS - Venue</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="container">
+    <!-- Start Navbar Section -->
+    <nav class="navbar navbar-expand-md navbar-light">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+        aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="landing.html">Home </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="agenda.html">Schedule</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="speakers.html">Speakers</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-success active" href="venue.html">Venue</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Contact</a>
+          </li>
+        </ul>
+        <div class="mt-2 mt-md-0">
+          <button class="btn btn-success ">Get a ticket!</button>
+        </div>
+      </div>
+    </nav>
+    <!-- End Navbar Section -->
+  </header>
+  <!-- Start Venue Section -->
+  <section class="container">
+    <div class="container">
+      <h3 class="text-center font-weight-bold">Venue</h3>
+    </div>
+    <div class="text-center">
+      <p><span class="text-success font-weight-bold">Poznań University of Technology</span><br>
+        Plac Marii Skłodowskiej-Curie 5, 60-965 Poznań
+      </p>
+    </div>
+
+    <div class="row text-center">
+      <div class="col-sm-6">
+        <img class="img-fluid media-body" src="img/venuepic.png" height="auto" width="90%">
```

Comment:
przytnij mapę i zdjęcie by były dokładnie w tym samym formacie np. 4:3 ;) wtedy to lepiej wygląda